### ISSUE TITLE
toc position & affix

### ISSUE THREAD

**Issue by lattekun** [LotarProject/dokuwiki-template-bootstrap3#68]
1. I want to locate toc in sidebar when I access wiki with desktop or tablet pc.
And when I access wiki with android or iPhone, I want to locate toc left of right side.
How can I revise file? (main.php or bootstrap.css etc...)

2. I want to fix sidebar when I scroll with mouse.
Using 'affix' function? How can I revise file?

3. I`m korean. In sidebar, letter is broken.
Do I need to set up a php file to utf-8?

My English is a lack, but thank you to read my problem.

**Comment by mluettecke** [comment]
**1 and 2**
Do I understand you right that you want the table of contents in the sidebar instead of the sidebar on big screens? And I think you mean on smaller screenns the toc shall not be in the sidebar?

I made a local branch of the template and changed exactly this. My toc is in the sidebar and also fixed while scrolling.

@LotarProject Would there be interest to add this feature if I made a branch?
